Index.html

```diff
@@ -0,0 +1,30 @@
+        <img src="./Images/hawa_mahal.jpg" alt="Jaipur">
+        <h2>Amer Fort</h2>
+        <p>Amer Fort or Amber Fort is a fort located in Amer, Rajasthan, India. Amer is a town with an area of 4 square kilometres (1.5 sq mi)located 11 kilometres (6.8 mi) from Jaipur, the capital of Rajasthan. Located high on a hill, it is the principal tourist attraction in Jaipur.Amer Fort is known for its artistic style elements. With its large ramparts and series of gates and cobbled paths, the fort overlooks Maota Lake,which is the main source of water for the Amer Palace.
+
+            Amer Palace is great example of Rajput architecture. Some of its buildings and work have influence of Mughal architecture.Constructed of red sandstone and marble, the attractive, opulent palace is laid out on four levels, each with a courtyard. It consists of the Diwan-e-Aam, or "Hall of Public Audience", the Diwan-e-Khas, or "Hall of Private Audience", the Sheesh Mahal (mirror palace), or Jai Mandir, and the Sukh Niwas where a cool climate is artificially created by winds that blow over a water cascade within the palace. Hence, the Amer Fort is also popularly known as the Amer Palace.The palace was the residence of the Rajput Maharajas and their families. At the entrance to the palace near the fort's Ganesh Gate, there is a temple dedicated to Shila Devi, a Goddess of the Chaitanya cult, which was given to Raja Man Singh when he defeated the Raja of Jessore, Bengal in 1604. (Jessore is now in Bangladesh).[4][11][12] Raja Man Singh had 12 queens so he made 12 rooms, one for each Queen. Each room had a staircase connected to the King’s room but the Queens were not to go upstairs. Raja Jai Singh had only one queen so he built one room equal to three old queen’s rooms.
+            
+            This palace, along with Jaigarh Fort, is located immediately above on the Cheel ka Teela (Hill of Eagles) of the same Aravalli range of hills. The palace and Jaigarh Fort are considered one complex, as the two are connected by a subterranean passage. This passage was meant as an escape route in times of war to enable the royal family members and others in the Amer Fort to shift to the more redoubtable Jaigarh Fort.Annual tourist visitation to the Amer Palace was reported by the Superintendent of the Department of Archaeology and Museums as 5000 visitors a day, with 1.4 million visitors during 2007.At the 37th session of the World Heritage Committee held in Phnom Penh, Cambodia, in 2013, Amer Fort, along with five other forts of Rajasthan, was declared a UNESCO World Heritage Site as part of the group Hill Forts of Rajasthan</p>
+        <img src="./Images/Amer_Fort.jpg" alt="Jaipur">
+    </ol>
+</body>
+</html>
```

Comment:
All the files should have a new line at the end.

It's standard to have a blank LF/CRLF as a file terminator. When you will end up working on production-grade applications, linters associated with such applications will end up complaining about such seemingly trivial concerns. It is important therefore that you make it a habit to write clean code from the start.

One [helpful ref.](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3) that gives a gist of why new lines are a must

Pls make a habit of working hand-in-hand with helpful tools like [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) & [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) that come along with the VSCode editor & configure them to run automatically on save.

It should save you from the hassle of manually making such or other formatting changes.